### PR TITLE
Hyperconvection Circuit Imprinter

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

adds a circuit imprinter that consumes less materials but makes heat, just like the other ones
## Why / Balance

sci might actually roll out more tech to crew now :clueless: 
## Media

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Piksqu & Boaz1111
- add: Added the hyperconvection circuit imprinter

### CODE REVIEW

**Comment by Boaz1111** [comment]
disregard this for now, need to see the test failure

File: Resources/Prototypes/Recipes/Lathes/electronics.yml [hunk 1]
```diff
@@ -277,8 +277,18 @@
...
-     Steel: 100
-     Glass: 900
+    Steel: 100
+    Glass: 900
+
+- type: latheRecipe
+  id: CircuitImprinterHyperConvectionMachineCircuitboard
+  result: CircuitImprinterHyperConvectionMachineCircuitboard
+  category: Circuitry
+  completetime: 4
```
Comment (TheShuEd) [inline Resources/Prototypes/Recipes/Lathes/electronics.yml:287]: need 
`materials:`

Comment (Boaz1111) [reply Resources/Prototypes/Recipes/Lathes/electronics.yml:287]: oh my god that's it tysm

**Comment by TheShuEd** [comment]
:trollface:

**Comment by Boaz1111** [comment]
ed what are you scheming about

**Comment by FungiFellow** [comment]
> :trollface:

I can't wait to hop Sci's Counter and request several hundred low cost items to superheated the room

**Comment by deltanedas** [comment]
> I can't wait to hop Sci's Counter and request several hundred low cost items to superheated the room

non issue if you make parallel radiator array and good cooling (i did this before its op)

**Comment by EmoGarbage404** [comment]
My main concern is that the circuit imprinter being such an everpresent sink for glass and gold would make this upgrade overly dominant in terms of upgrades. I think it may be more reasonable to first nerf the price of boards down into the 5/4 glass range, since this would be less obviously good

**Comment by vorkathbruh** [comment]
> > :trollface:
> 
> I can't wait to hop Sci's Counter and request several hundred low cost items to superheated the room

you can already do this though

**Comment by Boaz1111** [comment]
> My main concern is that the circuit imprinter being such an everpresent sink for glass and gold would make this upgrade overly dominant in terms of upgrades. I think it may be more reasonable to first nerf the price of boards down into the 5/4 glass range, since this would be less obviously good

i could do one of two things:
-make the upgrade less strong
-do what you suggested
your call

**Pull request merged**